src/main/kotlin/com/humlib/controller/HumansController.kt

```diff
@@ -0,0 +1,37 @@
+package com.humlib.controller
+
+import com.humlib.model.HumanProfile
+import com.humlib.repository.HumanProfileRepository
+import com.humlib.security.annotations.IsHumanWithSameId
+import org.springframework.security.core.Authentication
+import org.springframework.web.bind.annotation.*
+import java.util.*
+
+@RestController
+@RequestMapping("/humans/{id}")
+@IsHumanWithSameId
+class HumansController(
+    val humanProfileRepository: HumanProfileRepository
+) {
+
+    @GetMapping
+    fun getHuman(@PathVariable id: UUID, authentication: Authentication): HumanProfile {
```

Comment:
HumanProfile = Human? Wie ist das in der Fachlichkeit? Wollen wir hier "DTOs" ans FE übergeben, um unsere Fachlichkeit unabhängig von der "Präsentation" (also in dem Falle der API) zu machen?